tools/generator/test/test_languages.py

```diff
@@ -8,14 +8,18 @@
 import shutil
 import subprocess
 import unittest
+from sys import stderr
 
-from generator.test.utils import generate_player
+from generator.game import load_game
+from generator.player import check_player
+from generator.test.utils import generate_player, load_test_game
 
 
 class TestLanguages(unittest.TestCase):
```

Comment:
Could the `compile_language()` function here be refactored with the `check_compile()` function in player.py? Seems like there's some common behavior? (Tell me if you think it's not worth it.)

Reply:
Made an attempt to refactor, I guess it gives slightly worse output (no distinction for a failure in `make` or `make tar)

Reply:
Maybe `check_compile()` and `check_tar()`?